components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -480,4 +483,8 @@ - (void)setEnableRippleBehavior:(BOOL)enableRippleBehavior {
   [self.tableView reloadData];
 }
 
+- (CGFloat)mdc_currentElevation {
+  return 0;
```

Comment:
If action sheet uses bottom sheet, shouldn’t this be not 0?

Reply:
I’ve just checked and looks like this should be 16. What made you choose 0, and why didnt you use bottom sheets elevation API to get this number?

Reply:
1. _What made you choose 0?_
  I figured since the component doesn't have an elevation API it should return 0, wouldn't we expect client's or a theming extension to override its base elevation?
2. _Why didn't you use bottom sheets elevation API?_
  ActionSheet isn't aware of the BottomSheet controller, only the bottom sheet presentation controller and transition controller.